dataset_config/1000_genomes/bigquery.json

```diff
@@ -1,15 +1,53 @@
-// Configuration for BigQuery indexer
-//
-// primary_key:
-//   Name of BigQuery column that represents primary key. See
+// Configuration for BigQuery indexer. See
 //   https://github.com/DataBiosphere/data-explorer-indexers#overview
 //   for more information.
```

Comment:
nit Can there be a newline between lines 3 and 4 (see old bigquery.json)?

Reply:
Done.